testsuite/features/secondary/srv_timezone_feature

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+#  1) create a user and assing him a timezone different than the server's timezone
+#  2) test that the popups in some scheduling actions appear in users prefered timezone 
+#  3) some scheduler tests based on previous bugs are unavoidable
+
+#@sle_minion
+Feature: Correct timezone display 
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Create a new user in a timezone different than server's timezone
+    When I follow the left menu "Users > User List > Active"
+    And I follow "Create User"
+    And I enter "user1" as "login"
+    And I enter "user1" as "desiredpassword"
+    And I enter "user1" as "desiredpasswordConfirm"
+    And I select "Mr." from "prefix"
+    And I enter "Test" as "firstNames"
+    And I enter "User" as "lastName"
+    And I enter "galaxy-noise@suse.de" as "email"
+    And I select "(GMT+0800) Malaysia" from "timezone"
+    And I click on "Create Login"
+
+  Scenario: Add roles
+    When I follow the left menu "Users > User List > Active"
+    And I follow "user1"
+    When the "role_satellite_admin" checkbox should be disabled
+    And I check "role_org_admin"
+    And I check "role_system_group_admin"
+    And I check "role_channel_admin"
+    And I check "role_activation_key_admin"
+    And I check "role_config_admin"
+    And I click on "Update"
+
+  Scenario: Login as the new user
+    Given I am authorized as "user1" with password "user1"
+    Then I should see a "user1" link
+
+# bsc 1192699, a P2 bug that is now fixed
```

Comment:
If it is fixed, I would not include the reference here.
We use to only have references to bugs that still opened and we workaround.

Reply:
No problem